packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.kt

```diff
@@ -1,48 +1,61 @@
 package expo.modules.updates.loader
 
-import android.content.Context
+import com.facebook.react.common.annotations.VisibleForTesting
```

Comment:
```suggestion
import androidx.annotation.VisibleForTesting
```

don't depend on react-native